test/README.md

```diff
@@ -0,0 +1,58 @@
+# Run integration tests from your local machine
+
+> This workflow  runs a single integration test from the local machine. It is a convenient alternate to GitHub actions; however, the limitation is that only one test can be run at a time.        
```

Comment:
If one test can be run at the time, then it won't difference where we can use `terraform apply`. So I don't think this would help us in improving the goal of running locally or solve the main problem here

Reply:
There are tools to run the github workflow locally, but the output is overwhelming. I guess I could try to write the logs to some text files for visibility, and it would just be an extension on top of integrationTest.yml